kubernetes/amd64-patch.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
```

Comment:
Same question as above. Why do we need to pin all the containers to amd64 nodes?

Reply:
answered above.